grunt_tasks/coffee_discovering_task.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function(grunt) {
```

Comment:
I would rather move the remaining task about coffee  and less in the cms-bundle, they are really needed for open-orchestra I think